app/Http/Controllers/Api/V1/UserController.php

```diff
@@ -6,11 +6,22 @@
 use App\Http\Resources\Api\V1\UserResource;
 use App\Models\User;
 use Illuminate\Support\Facades\Auth;
+use Illuminate\Http\Request;
 
 class UserController extends Controller
 {
     public function show()
     {
         return new UserResource(User::find(Auth::id()));
     }
+
```

Comment:
need a custom request to validate what is sent to the user update.  if you are only updating select fields you can use request->only() to only pull the specified fields from the request.... then you only need to validate those fields.